### PR TITLE
Customer provides username on registration

### PR DESCRIPTION
**Pull request by DannySantos**
Installed Devise

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -16,6 +16,8 @@
 	</header>
 
 	<main class="container">
+    <p class="notice"><%= notice %></p>
```
Comment (dmgarland) [inline app/views/layouts/application.html.erb]: Indentation!

File: db/migrate/20160721103649_add_industry_to_profile.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddIndustryToProfile < ActiveRecord::Migration
+  def change
+    add_column :profiles, :industry, :string
```
Comment (dmgarland) [inline db/migrate/20160721103649_add_industry_to_profile.rb]: Was this missing from somewhere else?

File: features/registration.feature [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Feature: A Customer Registers
+
+  @wip
```
Comment (dmgarland) [inline features/registration.feature]: 1 in 1 out :)

File: features/step_definitions/registration_steps.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+When(/^they fill in the form with valid details$/) do
+  fill_in "Username", with: "Finn"
+  fill_in "Email", with: "finn@finn.com"
+  fill_in "Password", with: "password"
+  fill_in "Confirm Password", with: "password"
+  click_button "Sign up"
+end
+
+Then(/^a Customer should be created$/) do
+  expect(Customer.count).to eq(1)
```
Comment (dmgarland) [inline features/step_definitions/registration_steps.rb]: I'm not sure this is adequate. If you add a check to see whether `expect(Customer.first.username).to eq('Finn')` I reckon you might not see anything... also so far I don't see any code that actually lets you sign in with a username, that would still be email by the looks of things. You need to subclass the sessions controller and change it's behaviour slightly.

File: db/migrate/20160721103649_add_industry_to_profile.rb (hunk 2 above)
Comment (DannySantos) [inline db/migrate/20160721103649_add_industry_to_profile.rb]: Yeah this was supposed to be part of a different pull request. Any way to remove it?

Comment (dmgarland) [inline db/migrate/20160721103649_add_industry_to_profile.rb]: 1) Checkout the branch where you need to put it back in (I'll call it branch A)
2) `git checkout customer_username db/migrate/20160721103649_add_industry_to_profile.rb`
3) `git add` and `git commit` in Branch A
4) Checkout customer_username again, `git rm` this file and then `git push`
:+1:

File: features/step_definitions/registration_steps.rb (hunk 4 above)
Comment (dmgarland) [inline features/step_definitions/registration_steps.rb]: See also: http://stackoverflow.com/questions/4633077/how-to-use-different-authentication-keys-for-two-devise-models
